### PR TITLE
Let's Encrypt certificates are issued by R3, R4, E1 or E2 now

### PR DESCRIPTION
**Pull request by jaykay-design**
See https://letsencrypt.org/certificates/

### CODE REVIEW

**Comment by automationsynd** [comment]
When will this PR be merged? I have certificates within the default 22 days before expiration that are not getting renewed because the current cert is issued by "R3" and the renew job is still looking for one containing "Let's Encrypt".

**Comment by automationsynd** [comment]
@jaykay-design any idea how soon this will be merged?

**Comment by jaykay-design** [comment]
@automationsynd, nope. Not my repository so I can't do anything.

You can use the [release I did](https://github.com/jaykay-design/letsencrypt-siteextension/releases/tag/1.0.6) on my PR branch if you want.

**Comment by joelnotified** [comment]
@jaykay-design looks like I've run into the same issue. All of my certs have stopped renewing, so thanks for this PR! Let's hope it gets merged and released soon.

In the meantime, do you know how to install an App Service extension from your zip release? Is this the way (manually adding it to a folder in Kudu) https://thefreezeteam.com/manually-add-extension-in-azure-app-service/?

**Comment by jaykay-design** [comment]
> In the meantime, do you know how to install an App Service extension from your zip release? Is this the way (manually adding it to a folder in Kudu) https://thefreezeteam.com/manually-add-extension-in-azure-app-service/?

The ZIP release only contains the source code of the project and not any executable so before being able to update the SiteExtension you would have to build the project first. Let me know if you need any help with that.

Once you have built the project you can just replace the files of the SiteExtensions similar as described in the link you provided.

**Comment by joelnotified** [comment]
@jaykay-design Alright! Thanks

File: LetsEncrypt.SiteExtension.Core/CertificateManager.cs [hunk 1]
```diff
@@ -138,7 +138,11 @@ public async Task<List<CertificateInstallModel>> RenewCertificate(bool skipInsta
                 body = await response.Content.ReadAsStringAsync();
                 IEnumerable<Certificate> certs = ExtractCertificates(body);
 
-                var expiringCerts = certs.Where(s => s.ExpirationDate < DateTime.UtcNow.AddDays(renewXNumberOfDaysBeforeExpiration) && (s.Issuer.Contains("Let's Encrypt") || s.Issuer.Contains("Fake LE")));
+
+                var issuerNames = new[] { "Fake LE", "Let's Encrypt", "R3", "R4", "E1", "E2" };
+                var letsEncryptCerts = ExtractCertificates(body).Where(s => issuerNames.Any(i => s.Issuer.StartsWith(i, StringComparison.InvariantCulture)));
```
Comment (joelnotified) [inline LetsEncrypt.SiteExtension.Core/CertificateManager.cs:143]: Just wondering about this change here @jaykay-design . `letsEncryptCerts` is never used right? 

I guess it doesn't really matter, but just wanted to let you know.

**Comment by jaykay-design** [comment]
Yes, you are right. Sorry for that.

**Pull request merged**